dll/win32/shell32/shldataobject.cpp

```diff
@@ -106,6 +106,8 @@ PIDLIST_ABSOLUTE SHELL_CIDA_ILCloneFull(_In_ const CIDA *pCIDA, _In_ UINT Index)
 
 PIDLIST_ABSOLUTE SHELL_DataObject_ILCloneFullItem(_In_ IDataObject *pDO, _In_ UINT Index)
 {
+    if (!pDO)
```

Comment:
In which circumstances was the SHELL_DataObject_ILCloneFullItem() function called with a NULL pDO? As far as I can see (pDO not being optional), the callers of that function should have checked for pDO being NULL themselves.

Reply:
`CCopyMoveToMenu::Initialize` already called it without checking the pointer. This PR also calls it without checking the pointer.